### PR TITLE
Review of CPU temperature article

### PR DESCRIPTION
**Pull request by dariavladykina**
### Description

Describe the overall goals of this pull request.

### CODE REVIEW

File: articles/CPUTemp.asm.xml [hunk 1]
```diff
@@ -31,24 +42,39 @@
...
+        <productname version="15-SP4">&sled;</productname>
+        <productname version="15-SP5">&sled;</productname>
+        <productname version="12-SP5">&sles4sap;</productname>
+        <productname version="15-SP4">&sles4sap;</productname>
+        <productname version="15-SP5">&sles4sap;</productname>
+        <productname version="12-SP5">&sleha;</productname>
+        <productname version="15-SP4">&sleha;</productname>
+        <productname version="15-SP5">&sleha;</productname>
+        <productname version="12-SP5">&slehpc;</productname>
+        <productname version="15-SP5">&slert;</productname>
```
Comment (janajaeger) [inline articles/CPUTemp.asm.xml:68]: What's the reasoning behind these seemingly random versions?

File: references/cpu-troubleshoot.xml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  <para>
+    If the CPU temperature is too high, here are actions you can take:</para>
+    <itemizedlist>
+       <listitem>
+        <para>Verify that the CPU cooling system, such as the fan or heat sink, is working correctly.
+        Ensure that the fan is spinning properly and that the heat sink is making proper contact with the CPU.
+        If necessary, you may need to replace the thermal paste between the CPU and the heat sink to improve heat transfer.</para>
+      </listitem>
+    <listitem>
+    <para>Adjust the power settings on your system to reduce heat generation. Lowering the CPU
```
Comment (janajaeger) [inline references/cpu-troubleshoot.xml:117]: This is kept obscure by design, isn't it? Or are there some general guidelines on how to do that for SUSE products? Maybe there is an article in the Support KB?

File: tasks/cpu-temp-check-realtime.xml
Comment (janajaeger) [inline tasks/cpu-temp-check-realtime.xml:1]: What's the difference exactly between sensors and watch and why would I want to have both? That doesn't come across quite as it should. It becomes clear in the last para, but that's a tad too late.

Comment (janajaeger) [inline tasks/cpu-temp-check-realtime.xml:1]: Check all the commands in your article for availability across platforms. Pretty sure we don't have ACPI on the mainframes, but other than that ... Add info blocks where needed.

File: tasks/cpu-temp-check.xml [hunk 3]
```diff
@@ -153,40 +77,78 @@ Sep 10 16:57:55 edison systemd[1]: Started Initialize hardware monitoring sensor
...
+ Core 5:        +40.0°C  (high = +82.0°C, crit = +100.0°C)</screen>
+ <calloutlist>
+  <callout arearefs="co-nvme-chip">
+    <para>
+    Specific hardware component or sensor chip being monitored.
+    </para>
+  </callout>
+  <callout arearefs="co-adapter-label">
+    <para>
+    Label, the descriptive name for the specific sensor on the chip.
```
Comment (janajaeger) [inline tasks/cpu-temp-check.xml:117]: Ditch the "Label"

**Review by janajaeger — COMMENTED** [review]
Overall a very nice job! Much better than the old version. However, there are a few things to note apart from the comments I added in the files:

- inconsistent file and id names (cpu-temp-*, but cpu-temperature-*, no temp* in both troubleshooting and more info). Please adjust both files and IDs and don't forget to adjust the assembly accordingly.
- assembly name doesn't follow our conventions. Please adjust. Check with the Cheatsheet if unsure.
- run a final xmlformat

File: articles/CPUTemp.asm.xml (hunk 1 above)
Comment (tomschr) [reply articles/CPUTemp.asm.xml:68]: This topic was discussed in our SLE writer's call today. We haven't completely resolved it so we postpone it when everybody is back from vacation.

File: references/.conkyrc
[the remark this answers is not in the document]
Comment (tomschr) [reply references/.conkyrc:1]: Probably something for the `.gitignore` file. :wink:

**Review by tomschr — COMMENTED** [review]
Some smaller comments. Overall, great job, Daria!

File: tasks/cpu-temp-check.xml (hunk 3 above)
Comment (dariavladykina) [inline tasks/cpu-temp-check.xml:117]:
```suggestion
    The descriptive name for the specific sensor on the chip.
```

**Review by dariavladykina — COMMENTED** [review]
Applied feedback

File: tasks/cpu-temp-check-realtime.xml
Comment (dariavladykina) [reply tasks/cpu-temp-check-realtime.xml:1]: sensors gives a snapshot of current temperatures while watch monitors them in real time and opens a separate screen for that; also, it doesn't stop until you tell it to. This is the difference.

File: articles/cpu-check-temperature.asm.xml [hunk 4]
```diff
@@ -31,24 +43,39 @@
...
+        <productname version="15-SP4">&sles4sap;</productname>
+        <productname version="15-SP5">&sles4sap;</productname>
+        <productname version="12-SP5">&sleha;</productname>
+        <productname version="15-SP4">&sleha;</productname>
+        <productname version="15-SP5">&sleha;</productname>
+        <productname version="12-SP5">&slehpc;</productname>
+        <productname version="15-SP5">&slert;</productname>
       </meta>
       <!-- SEO and social media -->
       <meta name="title" its:translate="yes">How to check CPU temperature</meta>
```
Comment (dariavladykina) [inline articles/cpu-check-temperature.asm.xml:72]:
```suggestion
      <meta name="title" its:translate="yes">Check and monitor CPU temperature</meta>
```

File: tasks/cpu-temperature-check.xml [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+
+<article xml:id="task-check-cpu-temperature" xml:lang="en"
+ role="task"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:its="http://www.w3.org/2005/11/its"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+
+<title>Working with hardware sensors</title>
```
Comment (janajaeger) [inline tasks/cpu-temperature-check.xml:22]: Pretending to be a reader who has no clue about all of this, this title doesn' tell me much. Actually, this topic has a structural problem as well. Rather make it two separate topics.

The first one: "Installing and configuring temperature sensors" containing all the stuff up to "Getting the temperature data"

The second one would be what's now section 2.1

Also, make sure you assign topic as the root element.

**Review by janajaeger — COMMENTED** [review]
Really nice article.
Still found a few issues with file names and IDs and one structural problem.
Also, article is just reserved for the assembly itself. Anything below that is a topic ;)

**Review by janajaeger — APPROVED** [review]
Well done. Get Toms' approval and merge!

File: articles/cpu-check-temperature.asm.xml [hunk 6]
```diff
@@ -49,25 +47,7 @@
...
-        <productname version="15-SP4">&sleha;</productname>
-        <productname version="15-SP5">&sleha;</productname>
-        <productname version="12-SP5">&slehpc;</productname>
-        <productname version="15-SP5">&slert;</productname>
+      <meta name="productname"><productname version="1">&alp;</productname><productname version="15-SP5">&sles;</productname><productname version="15-SP4">&sles;</productname><productname version="15-SP3">&sles;</productname><productname version="15-SP2">&sles;</productname><productname version="15-SP1">&sles;</productname><productname version="15-GA">&sles;</productname><productname version="12-SP5">&sles;</productname><productname version="15-SP4">&sled;</productname><productname version="15-SP5">&sled;</productname><productname version="12-SP5">&sles4sap;</productname><productname version="15-SP4">&sles4sap;</productname><productname version="15-SP5">&sles4sap;</productname><productname version="12-SP5">&sleha;</productname><productname version="15-SP4">&sleha;</productname><productname version="15-SP5">&sleha;</productname><productname version="12-SP5">&slehpc;</productname><productname version="15-SP5">&slert;</productname>
```
Comment (tomschr) [inline articles/cpu-check-temperature.asm.xml:50]: That looks quite unreadable. :wink: Nitpick: could we format it nicely (indented)?

**Pull request merged**